app/components/documents/document-card.js

```diff
@@ -311,4 +365,11 @@ export default class DocumentsDocumentCardComponent extends Component {
     }
     return false;
   }
+
+  @action
+  async cancelEditPiece() {
+    await this.loadPieceRelatedData.perform();
```

Comment:
this `loadPieceRelatedData` already contains the `loadSignatureRelatedData`.
Just noticed that now.
It might be better to just load that one. Will adjust the other ones I just added